app/src/main/java/com/example/composetemplate/navigation/Graph.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.composetemplate.navigation
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Graph interface represents a navigation graph within the application.
+ * A navigation graph is a collection of routes and navigation actions that define a complete navigation flow.
+ */
+interface Graph
+
+@Serializable
+sealed class Graphs {
```

Comment:
check if you can do it as a File instead of sealed class to avoid confuse and stuff like Garpf.EbtryGrapf, etc...